pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -202,6 +203,15 @@ func (c *Cluster) makeMgrDaemonContainer(mgrConfig *mgrConfig) v1.Container {
 			c.cephMgrOrchestratorModuleEnvs()...,
 		),
 		Resources: c.resources,
+		LivenessProbe: &v1.Probe{
+			Handler: v1.Handler{
+				HTTPGet: &v1.HTTPGetAction{
+					Path: "/",
+					Port: intstr.FromInt(metricsPort),
+				},
+			},
+			InitialDelaySeconds: 5,
+		},
```

Comment:
How long until the mgr will be restarted if a liveness probe fails? Just curious about the defaults.

Reply:
it would be killed immediately. There are other options that we can also set: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes

Reply:
Ok, the defaults seem sufficient thanks